krabs/krabs/trace.hpp

```diff
@@ -282,6 +282,20 @@ namespace krabs {
          */
         void set_default_event_callback(c_provider_callback callback);
 
+        /**
+         * <summary>
+         * Ignore Classic/MOF events that require calling TDH to get the provider which case be slow.
+         * Default behavior will leave MOF events enabled.
+         * </summary>
+         *
+         * <param name="ignore_mof_events">true to ignore MOF events</param>
+         * <example>
+         *    krabs::trace trace;
+         *    trace.set_ignore_mof_events(true);
+         * </example>
+         */
+        void set_ignore_mof_events(bool ignore_mof_events);
```

Comment:
Let's call this `set_mof_event_processing_enabled(bool mof_events_enabled)` with a default of `true` to avoid a double-negative.